packages/components/tailwind.config.js

```diff
@@ -3,7 +3,11 @@ const tokens = require("@chanzuckerberg/czedi-kit-tokens/json/variables-nested.j
 module.exports = {
   theme: {
     colors: tokens.eds.color,
-    extend: {},
+    fontFamily: {
+      sans: "Arimo, sans-serif",
```

Comment:
note that this isn't actually imported to storybook yet, I'm thinking I should create a global style file and include it there -- that can also be the file that's exported to Traject?

Reply:
alright ... here's my attempt at that: https://github.com/chanzuckerberg/lp-design-system/pull/231

Reply:
I think I'd be more comfortable calling this `arimo`